cluster-operator/src/test/java/io/strimzi/operator/cluster/MainIntegrationTest.java

```diff
@@ -28,7 +28,7 @@
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 
 @ExtendWith(VertxExtension.class)
-public class MainIT {
+public class MainIntegrationTest {
```

Comment:
I don't get why did you renamed it.

Reply:
I find having the `Test` suffix helps a bit with filtering and searching the code base.
I've reverted the change. Regardless I should not have include renames in this PR as they are not cleanups like the other changes.

Reply:
If you want, we can discuss that maybe on some call? We use `*IT` across the whole codebase. We also use the `*ST*` pattern for system tests. The name is also coded in the Failsafe Maven plugin configuration. So if nothing else, we should use the same naming everywhere. Personally, I like the `IT` because it makes it easy to distinguish them in IDE, but I don't think I'm too fixated on it.